src/scenes/Auth/Auth.tsx

```diff
@@ -11,12 +11,30 @@ import GalleryRedirect from 'scenes/_Router/GalleryRedirect';
 import breakpoints from 'components/core/breakpoints';
 import Spacer from 'components/core/Spacer/Spacer';
 
+// Preloading images for the welcome screen
+import { animatedImages } from 'src/scenes/WelcomeAnimation/Images';
+
+const preloadImages = () => {
+  animatedImages.forEach((image) => {
+    const img = new Image();
+    img.src = image.src ?? '';
```

Comment:
```
animatedImages.forEach((image = '') => {
```

Reply:
^ this should remove the need for nullish ??

Reply:
`image.src` is what we're checking not `image`, so I think we would need to destructure in the forEach? not totally sure

Reply:
nope, what you did is totally right! thanks!